frontend/src/components/phones/dashboard/PhoneDashboard.module.scss

```
@@ -0,0 +1,264 @@
+    padding-right: $spacing-md;
+    padding-bottom: $spacing-md;
+    border-bottom: 1px solid $color-light-gray-20;
+    margin: 0;
+
+    &:last-child {
+      border-bottom: none;
+    }
+
+    @media screen and #{$mq-sm} {
+      flex: 1;
+      justify-content: center;
+      align-items: center;
+      flex-direction: column;
+      text-align: center;
+      border-radius: $border-radius-md;
+      background: $color-light-gray-10;
+      padding: $spacing-md $spacing-lg;
+    }
+
+    .phone-statistics-title {
+      font-weight: 600;
+      border-radius: $border-radius-md;
+
+      @include text-title-3xs;
+
+      @media screen and #{$mq-sm} {
+        @include text-title-2xs;
+        @include font-firefox;
+      }
+    }
+
+    .phone-statistics-body {
+      line-height: 120%;
+
+      @media screen and #{$mq-sm} {
+        @include text-body-xs;
+      }
+    }
+  }
+
+  .inactive-statistics {
+    color: $color-light-gray-70;
+  }
+
+  .copy-icon {
+    padding: 0 $spacing-sm;
+    opacity: 0.5;
+    display: inline-block;
+    color: $color-light-gray-90;
+    align-self: center;
+  }
+
+  .copy-button-wrapper {
+    display: flex;
+    flex-direction: row;
+    gap: $spacing-xs;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+
+    .copy-button {
+      align-content: center;
+      justify-content: center;
+      display: flex;
+      appearance: none;
+      border: 0;
+      background-color: transparent;
+      text-align: center;
+      color: $color-grey-20;
+      cursor: pointer;
+
+      svg {
+        width: $layout-sm;
+      }
+
+      &:hover {
+        color: $color-grey-30;
+      }
+    }
+
+    .copied-confirmation {
+      @include text-body-sm;
+      opacity: 0;
+      padding: $spacing-xs $spacing-sm;
+      justify-content: center;
+      pointer-events: none;
+      transition: opacity 2s;
+      background-color: $color-violet-30;
+      color: $color-white;
+      border-radius: $border-radius-md;
+      position: absolute;
+      right: 0;
+      top: 50%;
+      transform: translateY(-50%);
+
+      &.is-shown {
+        pointer-events: auto;
+        opacity: 1;
+        // Don't fade in when appearing:
+        transition: opacity 0s;
+      }
+
+      @media screen and #{$mq-sm} {
+        position: static;
+        transform: none;
+      }
+    }
+  }
+
+  .phone-controls-container {
+    display: flex;
+    flex-direction: column;
+    margin-top: $spacing-md;
+    padding: 0 $spacing-lg $spacing-2xl $spacing-lg;
+    width: 100%;
+    gap: $spacing-lg;
+    border-bottom: 1px solid $color-light-gray-20;
+
+    @media screen and #{$mq-md} {
+      flex-direction: row;
+      width: auto;
+      gap: $layout-xl;
+      padding: $spacing-lg $spacing-lg $spacing-2xl $spacing-lg;
+      border-top: 1px solid $color-light-gray-20;
+    }
+
+    .phone-controls {
+      border-radius: 40px;
+      border: 2px solid $color-light-gray-20;
```

suggestion (non-blocking): 
```suggestion
      border: $border-radius-xs solid $color-light-gray-20;
```

---

Done!